client/src/ui-components/ClientOrdersComponent/ClientOrderTableRow.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from "react";
+import "./ClientOrderTableRow.css";
+function ClientOrderTableRow(props) {
+  const [status, setStatus] = useState(props.order.status);
+
+  const handleSubmit = () => {
+    if (status === "WAITING") {
+      setStatus("COMPLETED");
+    } else if (status === "COMPLETED") {
+      setStatus("WAITING");
+    }
+  };
+
+  return (
+    <tr>
+      <td style={{ width: "200px" }}>{props.order.id}</td>
+      <td style={{ width: "200px" }}>{props.order.totalPrice}€</td>
+      <td style={{ width: "200px" }}> {props.order.createdAt}</td>
+      {status === "WAITING" ? (
+        <td style={{ width: "300px", color: "#FF9E0A" }}>{status}</td>
+      ) : (
+        <td style={{ width: "300px", color: "green" }}>{status}</td>
+      )}
+
+      <td>
+        {" "}
+        <button onClick={handleSubmit}>Change Status</button>
+      </td>
+    </tr>
```

Comment:
It doesn't respect the GUI design:

-   Table rows and columns should be transparent
-   All component should have their own CSS file (button, td)